pom.xml

```diff
@@ -57,8 +57,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.7.0</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>8</source>
```

Comment:
Please don't do this. We have people running WavesJ on 1.6

Reply:
done